index.html

```diff
@@ -227,5 +227,18 @@ <h2>Languajes
         </div>
     </div>
 </section>
+<section class="form-section">
+    <div class="form">
+        <h2 class="form-title">Contact me</h2>
+        <p>If you have an application you are interested in developing, a feature 
+            that you need built or a project that needs coding. I’d love to help with it </p>
+        <form class="form-media">
+            <input type="text" id="form-name" placeholder="Enter name" maxlength="30">
+            <input type="email" id="form-email" placeholder="Enter email">
+            <textarea id="form-message" name="message"  maxlength="500" placeholder="Write your message here"></textarea>
```

Comment:
Please add `required` property to all fields so a form is validated before submission